### PR TITLE
fix: Markdownlint cleanup for RU

### PR DESCRIPTION
**Pull request by nschonni**

### CODE REVIEW

**Comment by nschonni** [comment]
@sashasushko not sure if you can take a look at this sometime. I can also split it up a bit if it's too big

**Comment by sashasushko** [comment]
@nschonni PR with 100 files or fewer is acceptable for me, thanks ❤️

File: files/ru/mdn/tools/kumascript/troubleshooting/index.md [hunk 1]
```diff
@@ -23,9 +23,9 @@ original_slug: MDN/Kuma/Troubleshooting_KumaScript_errors
 Необходимо проверить:
 
 - Использование фигурных скобок без намерения вызвать макрос.
-  - : Если вам нужно написать **\\{** в документе, не вызывая макрос, вы можете экранировать его используя _\\_ таким образом `\\{`
+  - : Если вам нужно написать \\{ в документе, не вызывая макрос, вы можете экранировать его используя таким образом `\\{`
 - Использование специального символа в параметрах макроса.
-  - : Если вам нужно использовать **"** или **\\\*\*** \*\*внутри параметра макроса, они могут быть экранированы с помощью `\\` или `\"`
+  - : Если вам нужно использовать " или \ внутри параметра макроса, они могут быть экранированы с помощью `\\` или `\"`
```
Comment (sashasushko) [inline files/ru/mdn/tools/kumascript/troubleshooting/index.md:28]:
```suggestion
  - : Если вам нужно использовать " или \ внутри параметра макроса, они могут быть экранированы с помощью `\"` и `\\` соответственно 
```

File: files/ru/web/api/canvasrenderingcontext2d/index.md [hunk 2]
```diff
@@ -127,7 +127,7 @@ ctx.fillRect(10, 10, 55, 50);
 - {{domxref("CanvasRenderingContext2D.moveTo()")}}
   - : Перемещает начальную точку нового фрагмента контура в точку с координатами **(x, y)**.
 - {{domxref("CanvasRenderingContext2D.lineTo()")}}
-  - : Соединяет последнюю точку фрагмента контура и точку с координатами `(x,y) `прямой линией .
+  - : Соединяет последнюю точку фрагмента контура и точку с координатами `(x,y)`прямой линией .
```
Comment (sashasushko) [inline files/ru/web/api/canvasrenderingcontext2d/index.md:130]:
```suggestion
  - : Соединяет последнюю точку фрагмента контура и точку с координатами `(x,y)` прямой линией.
```

**Pull request merged**